src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx

```diff
@@ -108,6 +108,13 @@ export default class ReviewCardField extends React.Component {
       : uiSchema['ui:title'];
   };
 
+  getSubtitle = () => {
```

Comment:
I think I like this. Can you add a test or two for it?

Reply:
Will do